### ISSUE TITLE
EXIF processing/marshalling fails with "unsupported value: +Inf"

### ISSUE THREAD

**Issue by mfg92** [gohugoio/hugo#12741]
### Description

I'm the maintainer of [mfg92/hugo-shortcode-gallery](https://github.com/mfg92/hugo-shortcode-gallery). I have noticed that starting with Hugo 1.30.0 or 1.31.0 the EXIF data of some images can not be processed anymore. I'm getting this error:
```
...
execute of template failed at <$original.Exif>:
error calling Exif:
metadata init failed:
json:
error calling MarshalJSON for type exif.Tags:
unsupported value: +Inf 
```

The line that causes this issue on my side is [this one](https://github.com/mfg92/hugo-shortcode-gallery/blob/master/layouts/shortcodes/gallery.html#L110).

My _hugo.toml_ contains these two lines (if that matters):
```TOML
[imaging.exif]
    includeFields = ".*"
```

This is the image that causes this problem, it is a focus stack of multiple images, created using _Helicon Focus 8.2.0_ and edited using _Darktable 4.6.1_:
![2023-05-06-LB1A1860-1879_Stacked(C,Smoothing4)-Jumping spider](https://github.com/user-attachments/assets/67de12c8-e01c-4f2f-ad94-63a2158c1740)

I'm assuming this has something to do with the switch of the EXIF library (72ff937e11a6375da5b13788f855eafcc2452b85).

### What version of Hugo are you using (`hugo version`)?

0.131.0

### Does this issue reproduce with the latest release?

yes

**Comment by mfg92** [comment]
Many thanks!

**Comment by alexvanderberkel** [comment]
@bep Is this fix already in version hugo version 0.133.0 reflected as I get a similar error while using the shortcode of @mfg92 

`.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/mfg92/hugo-shortcode-gallery@v1.3.0/layouts/shortcodes/gallery.html:122:21": execute of template failed: template: shortcodes/gallery.html:122:21: executing "shortcodes/gallery.html" at <$original.Exif>: error calling Exif: metadata init failed: json: error calling MarshalJSON for type exif.Tags: json: unsupported value: +Inf
`
